mytoyota/models/vehicle.py

```diff
@@ -226,19 +227,27 @@ def lock_status(self) -> Optional[LockStatus]:
             self._endpoint_data["status"] if "status" in self._endpoint_data else None
         )
 
-    async def get_summary(
+    async def get_summary(  # noqa: PLR0912, PLR0915
```

Comment:
Please do not simply use `# noqa: PLR0912, PLR0915` here. Instead, we should try to improve the code quality as described.